src/main/java/com/rakovets/course/java/core/practice/reflection/ReflectionUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.rakovets.course.java.core.practice.reflection;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ReflectionUtils {
+    public static Field getField(Class<?> clazz, String fieldName) throws NoSuchFieldException {
+        return clazz.getDeclaredField(fieldName);
+    }
```

Comment:
You inform the client about exception handling problems. This is bad for housing and communal services.